### PR TITLE
Adds Storage Container Animations

### PR DESCRIPTION
**Pull request by alsoandanswer**
dreamseeker_uSLitMhe1a

  - imageadd: "Adds a new animation for using storage objects."
  - rscadd: "Sorts generic bags, new sprites (plant bag, sheet bag, mining satchel, chem bag)."
  - tweak: "Removes self message for putting things into storage. (You put the X into Y message)."
 
Also fixes the manually mapped in TCFL crates.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/storage.dm [hunk 1]
```diff
@@ -42,6 +41,7 @@
 	var/use_sound = /singleton/sound_category/rustle_sound	//sound played when used. null for no sound.
 	var/list/starts_with // for pre-filled items
 	var/empty_delay = 0 SECOND // time it takes to empty bag. this is multiplies by number of objects stored
+	var/animated = TRUE /// whether or not we should have those cute little animations
```
Comment (FluffyGhoster) [inline code/game/objects/items/weapons/storage/storage.dm:44]:
```suggestion
	///Boolean, whether or not we should have those cute little animations
	var/animated = TRUE
```

File: code/game/objects/items/weapons/storage/bags.dm [hunk 2]
```diff
@@ -124,12 +119,9 @@
 
 /obj/item/storage/bag/plants
 	name = "plant bag"
-	icon = 'icons/obj/hydroponics_machines.dmi'
+	desc = "For storing your stems, seeds, buds, and any other euphemism for marijuana."
```
Comment (DreamySkrell) [inline code/game/objects/items/weapons/storage/bags.dm:122]: Should references Ambrosia or whatever instead.

File: code/game/objects/items/weapons/storage/storage.dm [hunk 3]
```diff
@@ -42,6 +41,8 @@
 	var/use_sound = /singleton/sound_category/rustle_sound	//sound played when used. null for no sound.
 	var/list/starts_with // for pre-filled items
 	var/empty_delay = 0 SECOND // time it takes to empty bag. this is multiplies by number of objects stored
+	///Boolean, whether or not we should have those cute little animations
```
Comment (DreamySkrell) [inline code/game/objects/items/weapons/storage/storage.dm:44]: What animations? This should explain what is the "animation" in this context. That being the item taking-out or putting-in animation (?).

**Review by DreamySkrell — CHANGES_REQUESTED** [review]

**Comment by DreamySkrell** [comment]
Also, I think the PR title should have an actually descriptive name. I read "Much Ado About the Bagginses" and I thought we're adding Lord of the Rings or whatever.

Should be titled "Storage object animation" or something, idk.

File: code/game/objects/items/weapons/storage/storage.dm [hunk 4]
```diff
@@ -412,20 +415,21 @@
...
 
 		if(!prevent_warning)
 			for(var/mob/M in viewers(user, null))
-				if (M == usr)
-					to_chat(usr, "<span class='notice'>You put \the [W] into [src].</span>")
+				if(M == usr)
+					continue
 				else if (M in range(1)) //If someone is standing close enough, they can tell what it is...
-					M.show_message("<span class='notice'>\The [user] puts [W] into [src].</span>")
+					M.show_message(SPAN_NOTICE("\The [user] puts [W] into [src]."))
```
Comment (Generalcamo) [inline code/game/objects/items/weapons/storage/storage.dm:430]:
```suggestion
					M.show_message(SPAN_NOTICE("\The [user] puts [W] into [src]."))
```
This feels like it should be a visible message sent to everyone. Then we can have blind and self-checking.

File: code/game/objects/items/weapons/storage/bags.dm [hunk 5]
```diff
@@ -124,12 +119,9 @@
 
 /obj/item/storage/bag/plants
 	name = "plant bag"
-	icon = 'icons/obj/hydroponics_machines.dmi'
+	desc = "For storing your stems, seeds, buds, and any other euphemism illicit substances."
```
Comment (Generalcamo) [inline code/game/objects/items/weapons/storage/bags.dm:122]:
```suggestion
	desc = "For storing your stems, seeds, buds, and any other illicit substances."
```

**Review by Generalcamo — CHANGES_REQUESTED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**